samples/sensor/mcp9600/boards/nrf52840dongle_nrf52840.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020, SER Consulting LLC  / Steven Riedl
```

Comment:
Have you actually tested on this?  It's an unusual board to use for external connections.

If you don't need it, it might be better to drop this and just leave the DK variant of the same SOC.

Reply:
Yes, actually on this (of course you have to wire up an I2C port ) I actually have a custom board that I also use, but makes sense to remove this, mainly tested on the kw and nrfdk.

Reply:
Deleted, will be in next push.